mw-webapp/src/dataAccessLogic/getUsersPreview.ts

```diff
@@ -15,8 +14,12 @@ export const getUsersPreview = async (): Promise<UserPreview[]> => {
   const firstUser = usersDTO[0];
 
   const ownWays = firstUser.ownWays.map((ownWayUuid) => {
-    const ownWay: WayPreview = waysPreview
-      .find((elem) => elem.uuid === ownWayUuid) ?? {} as WayPreview;
+    const ownWay = waysPreview
+      .find((elem) => elem.uuid === ownWayUuid);
+
```

Comment:
Pls add TODO comment that we need use hashmap here

Reply:
Done